### ISSUE TITLE
Borders not being rendered?

### ISSUE THREAD

**Issue by jmjpro** [diegomura/react-pdf#127]
Upon a cursory reading of the code I see that borders are supported. Do they only work on certain elements?
```
const CognitiveFunctionOverview = ({name}) => {
  const styles = StyleSheet.create({
    style: {
      paddingRight: 10,
      borderWidth: 1,
      borderStyle: 'solid',
      borderColor: '#CCCCCC'
    }
  });

  return (
  <View style={styles.style}>
    <Text>
      CF1 {name}
    </Text>
  </View>
  )
};
```
No border is displayed.

**Comment by diegomura** [comment]
We don't have border support yet. The only thing we are doing is expanding the styles `border` and so on into `borderBottom`, `borderLeft`...

I think the way to do this should be by checking in `elements/Base` if there is a border attribute, and render rects, lines or whatever based on the possible combinations. The trickiest scenario would be having two or three borders (ex. left and bottom) with a radius. But we can left that for a future iteration. 

I'll try to implement this, but I'm going to vacations in 2 days, so I can't promise it 😅 . If you want and can do it, please feel free to do it!

**Comment by jmjpro** [comment]
I added some cases to the switch and I see that it's getting there, but it seems to have no effect within the yoga code:
```
      case 'borderTopWidth':
        console.log('setting top border', value);
        this.layout.setBorder(Yoga.EDGE_TOP, value);
        break;
      case 'borderLeftWidth':
        console.log('setting left border', value);
        this.layout.setBorder(Yoga.EDGE_LEFT, value);
        break;
      case 'borderBottomWidth':
        this.layout.setBorder(Yoga.EDGE_BOTTOM, value);
        break;
      case 'borderRightWidth':
        this.layout.setBorder(Yoga.EDGE_RIGHT, value);
        break;
```

**Comment by diegomura** [comment]
Thanks for tackle this. Unfortunately I cannot help right now because I'm on vacations 😅  I'll come back in a couple of days and resume this. Thanks again!

**Comment by jmjpro** [comment]
@diegomura are you still away on vacation? Looking for some direction here...

**Comment by diegomura** [comment]
I arrived home today haha. I look forward to start working on these issues soon this week!

**Comment by jmjpro** [comment]
great thanks!

**Comment by diegomura** [comment]
Fixed! You have borders support on v0.4.2 now
